charts/incubator/deemix/values.yaml

```diff
@@ -0,0 +1,32 @@
+image:
+  repository: tccr.io/truecharts/deemix
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:c38cf91d6c069630359e4368ef7f1a56b7d1df37893c87f3e3c59aaef05db2d0
+
+env:
+  UMASK_SET: "{{ .Values.env.UMASK }}"
+  INTPORT: "{{ .Values.service.main.ports.main.targetPort }}"
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
```

Comment:
After tests passing, you can comment out those. See if the app works and there are not permissions errors on logs.
(under Run chart-testing (install)).
If there is any errors, slowly back off permissions.

Unless of course you know before hand an app needs root perms or root fs access. 
Usually apps that use the PUID/PGID need root access tho.

(Keep this as a general advice, not specific for this app)

Reply:
> (under Run chart-testing (install)).
Sorry, how do I do this ? I can't see the option. Is this on github or in the dev container?

Reply:
Its one of the automated tests you need to watch.